src/pages/Bridge/BridgeModals/BridgeDisclaimerModal.tsx

```diff
@@ -17,26 +17,37 @@ export const BridgeDisclaimerModal = ({
   heading,
   text,
   disclaimerText
-}: BridgeDisclaimerModalProps) => (
-  <Modal isOpen={isOpen} onDismiss={onDismiss} maxHeight={90}>
-    <Wrapper>
-      <ArrowRightCircle strokeWidth={0.5} size={75} color="#0E9F6E" />
-      <TitleWrapper>
-        <TYPE.body fontSize="22px" fontWeight="500" color="text1" textAlign="center">
-          {heading}
-        </TYPE.body>
-      </TitleWrapper>
-      <TYPE.main mb="16px" fontSize="16px" fontWeight="600" color="#EBE9F8" textAlign="center" lineHeight="1.6">
-        {text}
-      </TYPE.main>
-      <TYPE.small mb="24px" textAlign="center" fontSize="14px" lineHeight="1.6">
-        {disclaimerText}
-        Would you like to proceed?
-      </TYPE.small>
-      <ButtonPrimary mb="12px" onClick={onConfirm}>
-        CONFIRM
-      </ButtonPrimary>
-      <ButtonSecondary onClick={onDismiss}>CANCEL</ButtonSecondary>
-    </Wrapper>
-  </Modal>
-)
+}: BridgeDisclaimerModalProps) => {
+  const [disableConfirm, setDisableConfirm] = useState(false)
```

Comment:
Also noticing similar pattern with bridge modals I went in through couple of them and think they can be consolidated into less components couple of them have just few extra properties
![Screenshot 2022-04-06 at 16 54 44](https://user-images.githubusercontent.com/33226956/162003981-a4fa249d-07d2-4d9d-a4a9-736631819b6a.png)

Reply:
done